io.thp.numptyphysics.appdata.xml

```diff
@@ -4,17 +4,15 @@
   <id type="desktop">numptyphysics.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0+</project_license>
-  <summary>A crayon-drawing based physics puzzle game</summary>
-  <description>
-    <p>
-      Harness gravity with your crayon and set about creating blocks, ramps,
-      levers, pulleys and whatever else you fancy to get the little red thing
-      to the little yellow thing.
-    </p>
-  </description>
+  <summary>Fun physics puzzle games</summary>
+  <summary xml:lang="pt_BR">Jogos nos quais a física é necessária para resolver quebra-cabeçasJogos nos quais a física é necessária para resolver quebra-cabeças</summary>
+  <summary xml:lang="es">Rompecabezas que utilizan tu conocimiento de la física</summary>
```

Comment:
`Rompecabezas` does not have a plural – ni tampoco está en plural; hence `utilizan` should be changed to `utiliza` :wink: